virtinst/guest.py

```diff
@@ -990,7 +992,11 @@ def _add_default_usb_controller(self):
             # For pseries, we always assume OS supports usb3
             if qemu_usb3:
                 usb3 = True
-
+        elif self.os.is_loongarch64():
+            # For machvirt, we always assume OS supports usb3
+            if (qemu_usb3 and
+                self.conn.support.conn_machvirt_pci_default()):
```

Comment:
`conn_machvirt_pci_default()` is used to check whether the current version of libvirt is one where the `aarch64/virt` machine type defaults to PCI instead of the previous default of virtio-mmio. It's not relevant to loongarch64, you just need to check `qemu_usb3` as in the pSeries branch right above. Please update the comment as well.